sv/tests/test_utils.py

```diff
@@ -428,3 +433,159 @@ def laboratoire_confirmation_label(self) -> Locator:
+        self.zone_infestee_rayon_unite_base_locator = "input[name='zoneinfestee_set-{}-unite_rayon'][value='{}']"
+        self.zone_infestee_surface_infestee_totale_base_locator = "#id_zoneinfestee_set-{}-surface_infestee_totale"
+        self.zone_infestee_surface_infestee_totale_unite_base_locator = (
+            "input[name='zoneinfestee_set-{}-unite_surface_infestee_totale'][value='{}']"
+        )
+        self.zone_infestee_total_forms = self.page.locator('input[name="zoneinfestee_set-TOTAL_FORMS"]')
+
+        # Boutons
+        self.add_zone_infestee_btn = page.get_by_role("button", name="Ajouter une zone infestée")
+        self.enregistrer = page.get_by_role("button", name="Enregistrer")
+
+    def _fill_zone_infestee_form(
+        self, index, zoneinfestee: ZoneInfestee, detections_zone_infestee: Optional[Tuple[FicheDetection, ...]] = None
+    ):
+        detections_zone_infestee = detections_zone_infestee or ()
+        self.page.locator(self.zone_infestee_nom_base_locator.format(index)).fill(zoneinfestee.nom)
+        self.page.locator(self.zone_infestee_rayon_base_locator.format(index)).fill(str(zoneinfestee.rayon))
+        self._select_unite_rayon_zone_infestee(zoneinfestee.unite_rayon, index)
+        self.page.locator(self.zone_infestee_surface_infestee_totale_base_locator.format(index)).fill(
+            str(zoneinfestee.surface_infestee_totale)
+        )
+        self._select_unite_surface_infestee_totale(zoneinfestee.unite_surface_infestee_totale, index)
+        self._select_detections_in_zone_infestee(index, detections_zone_infestee)
+
+    def _select_unite_rayon_zone_tampon(self, unite: FicheZoneDelimitee.UnitesRayon):
+        match unite:
+            case FicheZoneDelimitee.UnitesRayon.KILOMETRE:
+                self.rayon_zone_tampon_unite_km.click()
+            case FicheZoneDelimitee.UnitesRayon.METRE:
+                self.rayon_zone_tampon_unite_m.click()
+
+    def _select_unite_surface_tampon_totale(self, unite: FicheZoneDelimitee.UnitesSurfaceTamponTolale):
+        match unite:
+            case FicheZoneDelimitee.UnitesSurfaceTamponTolale.METRE_CARRE:
+                self.surface_tampon_totale_unite_m2.click()
+            case FicheZoneDelimitee.UnitesSurfaceTamponTolale.KILOMETRE_CARRE:
+                self.surface_tampon_totale_unite_km2.click()
+
+    def _select_unite_surface_infestee_totale(self, unite: ZoneInfestee.UnitesSurfaceInfesteeTotale, index: int):
+        match unite:
+            case ZoneInfestee.UnitesSurfaceInfesteeTotale.HECTARE:
+                self.page.locator(
+                    self.zone_infestee_surface_infestee_totale_unite_base_locator.format(index, "ha")
+                ).click(force=True)
+            case ZoneInfestee.UnitesSurfaceInfesteeTotale.METRE_CARRE:
+                self.page.locator(
+                    self.zone_infestee_surface_infestee_totale_unite_base_locator.format(index, "m2")
+                ).click(force=True)
```

Comment:
Pourquoi a ton besoin de force=True dans ces situations ?

Reply:
https://playwright.dev/docs/input#forcing-the-click
Dans mon cas, c'est le label qui intercepte le click.